src/lib/pages/types.ts

```diff
@@ -57,14 +57,26 @@ export namespace V2 {
   }
 
   export interface Page {
+    uuid?: string,
     title?: string,
     path?: string,
     seo?: Seo
     header_html?: string,
+    draft_status?: 'STAGED_DRAFT' | 'DRAFT'
   }
 
-  export type UpdatePageResponse = void;
-  export type DuplicatePageResponse = void;
+  export interface UpdatePage {
+    uuid?: string,
+    title: string,
+    path: string,
+    seo?: Seo,
+    header_html?: string,
+    draft_status?: 'STAGED_DRAFT' | 'DRAFT'
+  }
+
+  export type UpdatePageResponse = Page;
```

Comment:
These next few types will have to change to inherit `ExistingPage` if the above comments are followed.